controllers/grid/files/review/form/ManageReviewFilesForm.inc.php

```diff
@@ -90,6 +90,16 @@ function initData($args, $request) {
 	function execute($args, $request, $stageSubmissionFiles) {
 		parent::execute($args, $request, $stageSubmissionFiles, SUBMISSION_FILE_REVIEW_FILE);
 	}
+
+	/**
+	 * @copydoc ManageSubmissionFilesForm::importFile()
+	 */
+	protected function importFile($context, $submissionFile, $fileStage) {
+		parent::importFile($context, $submissionFile, $fileStage);
+
+		$submissionFileDao = DAORegistry::getDAO('SubmissionFileDAO');
+		$submissionFileDao->assignRevisionToReviewRound($newFileId, $newRevision, $this->getReviewRound());
```

Comment:
It was already doing that, but in a parent class that shouldn't have had to know about review implementation stuff. I moved it here as part of a clean-up.